src/status_im/contexts/wallet/data_store.cljs

```diff
@@ -309,3 +309,40 @@
   (-> collectible
       transforms/json->clj
       transform-collectible))
+
+(defn- bridge-amount-greater-than-bonder-fees?
```

Comment:
Just moved them from send.event to data-store file so i can used in swap as well.